lib/warder/runner.rb

```diff
@@ -21,10 +22,19 @@ def perform
     private
 
     def run_command
-      @stdout.puts "executing '#{command_with_options}'\n"
+      @stdout.puts(exec_msg) unless quiet?
       IO.popen(command_with_options).each do |line|
         yield(line)
       end
+      @stdout.puts(stats_msg) if stats?
+    end
+
+    def exec_msg
+      "executing '#{command_with_options}'\n"
+    end
+
+    def stats_msg
+      "found #{@issues.to_i} #{klass::CLI_FULL_OPTION.sub('-', ' ')} issues\n"
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.